### PR TITLE
5.17 with Windows

### LINKED ISSUE #4: Windows support?
Issue: No Windows support

Hi! I'm trying to get [altair_saver](https://anaconda.org/conda-forge/altair_saver) to work, but it depends on this package. Are there any plans on supporting Windows?
<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
| # Name             | Version     | Build             | Channel     |
|--------------------|-------------|-------------------|-------------|
| altair             | 4.1.0       | py_1              | conda-forge |
| altair_data_server | 0.4.1       | py_0              | conda-forge |
| altair_viewer      | 0.3.0       | py_0              | conda-forge |
| attrs              | 19.3.0      | py_0              | conda-forge |
| brotlipy           | 0.7.0       | py37h4ab8f01_1000 | conda-forge |
| ca-certificates    | 2020.4.5.2  | hecda079_0        | conda-forge |
| certifi            | 2020.4.5.2  | py37hc8dfbb8_0    | conda-forge |
| cffi               | 1.14.0      | py37ha419a9e_0    | conda-forge |
| cryptography       | 2.9.2       | py37h26f1ce3_0    | conda-forge |
| entrypoints        | 0.3         | py37hc8dfbb8_1001 | conda-forge |
| idna               | 2.9         | py_1              | conda-forge |
| importlib-metadata | 1.6.1       | py37hc8dfbb8_0    | conda-forge |
| importlib_metadata | 1.6.1       | 0                 | conda-forge |
| intel-openmp       | 2020.0      | 166               |             |
| jinja2             | 2.11.2      | pyh9f0ad1d_0      | conda-forge |
| jsonschema         | 3.2.0       | py37hc8dfbb8_1    | conda-forge |
| libblas            | 3.8.0       | 15_mkl            | conda-forge |
| libcblas           | 3.8.0       | 15_mkl            | conda-forge |
| liblapack          | 3.8.0       | 15_mkl            | conda-forge |
| markupsafe         | 1.1.1       | py37h8055547_1    | conda-forge |
| mkl                | 2020.0      | 166               |             |
| nodejs             | 14.4.0      | 0                 | conda-forge |
| numpy              | 1.18.5      | py37hae9e721_0    | conda-forge |
| openssl            | 1.1.1g      | he774522_0        | conda-forge |
| pandas             | 1.0.4       | py37h3bbf574_0    | conda-forge |
| pip                | 20.1.1      | py_1              | conda-forge |
| portpicker         | 1.3.1       | py37_0            |             |
| pycparser          | 2.20        | py_0              | conda-forge |
| pyopenssl          | 19.1.0      | py_1              | conda-forge |
| pyrsistent         | 0.16.0      | py37h8055547_0    | conda-forge |
| pysocks            | 1.7.1       | py37hc8dfbb8_1    | conda-forge |
| python             | 3.7.7       | h81c818b_4        |             |
| python-dateutil    | 2.8.1       | py_0              | conda-forge |
| python_abi         | 3.7         | 1_cp37m           | conda-forge |
| pytz               | 2020.1      | pyh9f0ad1d_0      | conda-forge |
| selenium           | 3.141.0     | py37h8055547_1001 | conda-forge |
| setuptools         | 47.1.1      | py37hc8dfbb8_0    | conda-forge |
| six                | 1.15.0      | pyh9f0ad1d_0      | conda-forge |
| sqlite             | 3.31.1      | h2a8f88b_1        |             |
| toolz              | 0.10.0      | py_0              | conda-forge |
| tornado            | 6.0.4       | py37hfa6e2cd_0    | conda-forge |
| urllib3            | 1.25.9      | py_0              | conda-forge |
| vc                 | 14.1        | h869be7e_1        | conda-forge |
| vs2015_runtime     | 14.16.27012 | h30e32a0_2        | conda-forge |
| wheel              | 0.34.2      | py_1              | conda-forge |
| win_inet_pton      | 1.1.0       | py37_0            | conda-forge |
| wincertstore       | 0.2         | py37_1003         | conda-forge |
| zipp               | 3.1.0       | py_0              | conda-forge |
| zlib               | 1.2.11      | h2fa13f4_1006     | conda-forge |
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
```
|     active environment : | altair                                                                  |
|-------------------------:|-------------------------------------------------------------------------|
|    active env location : | C:\Users\spsjo\anaconda3\envs\altair                                    |
|            shell level : | 2                                                                       |
|       user config file : | C:\Users\spsjo\.condarc                                                 |
| populated config files : | C:\Users\spsjo\.condarc                                                 |
|          conda version : | 4.8.3                                                                   |
|      conda-build version | : 3.18.11                                                               |
|         python version : | 3.7.7.final.0                                                           |
|       virtual packages : | __cuda=10.1                                                             |
|       base environment : | C:\Users\spsjo\anaconda3 (writable)                                     |
|           channel URLs : | https://conda.anaconda.org/conda-forge/win-64                           |
|                          | https://conda.anaconda.org/conda-forge/noarch                           |
|                          | https://repo.anaconda.com/pkgs/main/win-64                              |
|                          | https://repo.anaconda.com/pkgs/main/noarch                              |
|                          | https://repo.anaconda.com/pkgs/r/win-64                                 |
|                          | https://repo.anaconda.com/pkgs/r/noarch                                 |
|                          | https://repo.anaconda.com/pkgs/msys2/win-64                             |
|                          | https://repo.anaconda.com/pkgs/msys2/noarch                             |
|          package cache : | C:\Users\spsjo\anaconda3\pkgs                                           |
|                          | C:\Users\spsjo\.conda\pkgs                                              |
|                          | C:\Users\spsjo\AppData\Local\conda\conda\pkgs                           |
|       envs directories : | C:\Users\spsjo\anaconda3\envs                                           |
|                          | C:\Users\spsjo\.conda\envs                                              |
|                          | C:\Users\spsjo\AppData\Local\conda\conda\envs                           |
|               platform : | win-64                                                                  |
|             user-agent : | conda/4.8.3 requests/2.23.0 CPython/3.7.7 Windows/10 Windows/10.0.18362 |
|          administrator : | False                                                                   |
|             netrc file : | C:\Users\spsjo/.netrc                                                   |
|           offline mode : | False                                                                   |
</details>

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #7 
Fixes #6 
Fixes #3 
Fixes #4 

Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/vega-cli This is ready for review / merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+    - freetype  # [not win]
+    - giflib    # [not win]
+    - glib      # [not win]
+    - jpeg      # [not win]
+    - libpng    # [not win]
+    - pango     # [not win]
+    - yarn      # [not win]
   run:
-    - nodejs
+    - nodejs 14.*
```
Comment (jamesmyatt) [inline recipe/meta.yaml:28]: Why does this require nodejs 14.*?

Comment (xhochy) [reply recipe/meta.yaml:28]: This package includes node-canvas which is binary and thus pinned to the major release of `nodejs`. On Linux and MacOS we could build easily for more node versions but on Windows I wasn't able to get the build going and thus I'm using the pre-built node-canvas there (which is only available for node 14.*).

Comment (jamesmyatt) [reply recipe/meta.yaml:28]: OK. Thanks

**Pull request merged**